h2integrate/core/h2integrate_model.py

```diff
@@ -434,6 +434,7 @@ def create_finance_model(self):
                 "finance_groups", [default_finance_group_name]
             )
             tech_names = subgroup_params.get("technologies")
+            commodity_stream = subgroup_params.get("commodity_stream", None)
```

Comment:
I couldn't find any sort of docstring or comment explaining what `commodity_stream` actually is, if it's just a singular value, or if it could be a list of streams, etc. Could you please add that where relevant? Probably within the `specifiying_finance_parameters.md` as well?